src/backend/distributed/utils/listutils.c

```diff
@@ -155,6 +157,44 @@ ListToHashSet(List *itemList, Size keySize, bool isStringList)
 }
 
 
+/*
+ * ListAppendUniqueUint64 appends the given uint64 if it does not
+ * exist in the list.
+ */
+List *
+ListAppendUniqueUint64(List *list, uint64 datum)
```

Comment:
Is this code no longer used? It shows up as not covered.

Reply:
Yeah after changing the way we found jobids, this is no longer necessary.